tgui/src/interfaces/turbine_computer.ract

```diff
@@ -0,0 +1,51 @@
+<script>
+component.exports = {
+	data: {
+		powerState (status) {
+			switch (status) {
+				case 1: return 'good'
+				default: return 'bad'
+			}
+		}
+	}
+}
+</script>
+<ui-notice>
+	<ui-section label='Reconnect'>
+		<div style='float:right'>
+			<ui-button icon='refresh' action='reconnect' state='{{data.connected}}'>Reconnect</ui-button>
+		</div>
+	</ui-section>
+</ui-notice>
+<ui-display title='Turbine Controller'>
+	<ui-section label='Status'>
+		{{#if data.working}}
+			<span class='{{powerState(data.online)}}'>{{data.online && !(data.compressor_broke || data.turbine_broke) ? "Online" : "Offline"}}</span>
+		{{else}}
+			<span class='bad'>Broken</span>
+		{{/if}}
+		<div style='float:right'>
+			<ui-button icon='power-off' action='power' state='{{data.working}}' style='{{data.online ? "selected" : ""}}'>On</ui-button>
+			<ui-button icon='close' action='power' state='{{data.working}}' style='{{data.online ? "" : "selected"}}'>Off</ui-button>
+		</div>
+		{{#if data.compressor_broke}}
```

Comment:
Use `||` here instead of two ifs.

Reply:
2nd line should actually say Turbine instead, changing that.

Reply:
resolved